benchmarks/src/main/kotlin/chat/BenchmarkConfiguration.kt

```diff
@@ -0,0 +1,192 @@
+package chat
+
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.asCoroutineDispatcher
+import kotlinx.coroutines.channels.Channel
+import kotlinx.coroutines.scheduling.ExperimentalCoroutineDispatcher
+import org.apache.commons.math3.stat.descriptive.moment.StandardDeviation
+import java.util.concurrent.ForkJoinPool
+
+// Benchmarks configuration to monitor
+
+/**
+ * Underlying thread pool size for coroutines.
+ */
+val THREADS = listOf(4, 8, 16, 1)
+/**
+ * Chat users count.
+ */
+val USER_COUNT = listOf(10000)
+/**
+ * Maximum percentage of all chat users a user can be friends with.
+ */
+val MAX_FRIENDS_PERCENTAGE = listOf(0.2)
+/**
+ * Coroutines channels type.
+ */
+val CHANNEL_TYPES = listOf(ChannelType.UNLIMITED, ChannelType.RENDEZVOUS, ChannelType.BUFFERED_1, ChannelType.BUFFERED_16, ChannelType.BUFFERED_256)
+/**
+ * The average amount work that will be executed on CPU.
+ */
+val AVERAGE_WORK = listOf(80, 40)
+/**
+ * Implementations of users that contain different ways to find a user to send a message.
+ */
+val BENCHMARK_MODE = listOf(BenchmarkModes.USER_WITH_FRIENDS, BenchmarkModes.USER_WITHOUT_FRIENDS)
+/**
+ * Coroutines dispatcher types
+ */
+val DISPATCHER_TYPES = listOf(DispatcherTypes.EXPERIMENTAL, DispatcherTypes.FORK_JOIN)
+
+// Benchmark configuration that are not shown in the output file
+
+/**
+ * Execution time of each benchmark.
+ */
+const val BENCHMARK_TIME_MS = 2000L
```

Comment:
`Int` would be sufficient

Reply:
Yes, but Thread.sleep requires long. Should I cast Int to Long in Thread.sleep?